internal/services/recoveryservices/site_recovery_replicated_vm_resource_test.go

```diff
@@ -45,6 +45,21 @@ func TestAccSiteRecoveryReplicatedVm_des(t *testing.T) {
 	})
 }
 
+func TestAccSiteRecoveryReplicatedVm_targetDiskEncryption(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_site_recovery_replicated_vm", "test")
+	r := SiteRecoveryReplicatedVmResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.targetDiskEncryption(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
```

Comment:
Since it looks like it's possible to update the `disk_encryption_key` and `key_encryption_key` it would make sense to add an update step to this test as well?

Reply:
Hi @stephybun thanks for pointing out the issue, I've fixed the case issue related to the key vault permission, and actually this new test is waiting for `target_zone` property to be added by #15617 first.

For the update step, these properties are nested in `managed_disk` which is ForceNew, so it will not trigger the update